css/README.md

```diff
@@ -0,0 +1,45 @@
+# CSS Style Guide
```

Comment:
general thought - although it is not longer so much trouble as before - it might be good to define supported browsers (and "check" them - directly and/also with things like caniuse.com) - nowadays it is mostly mobile browsers that can lead to problems (and/or safari ;-))